tests.py

```diff
@@ -0,0 +1,15 @@
+from shorter import html_paragraphs
+import unittest
+from mock import MagicMock
+
+class ShorterModuleTest(unittest.TestCase):
+    def test_html_paragraphs(self):
+        soup = MagicMock(return_value=[
+            "paragraph1"
+            "paragraph2"
+            ])
+        function_test = html_paragraphs('https://www.crummy.com/software/BeautifulSoup/bs4/doc/')
```

Comment:
Disconnect your network and try to run UTs. They will fail. You are not mocking the network call.